utils/defs.tex

```diff
@@ -45,7 +45,6 @@
 \newcommand{\KEYWORD}[1]{\textit{#1}}
 %%
 \newcommand{\CorCpp}{\textit{C/C++}\xspace}
-\newcommand{\CorCppFor}{\textit{C/C++/Fortran}\xspace}
 \newcommand{\Fortran}[1][]{%
```

Comment:
Can/should we remove the Fortran macro?

Reply:
Since, we are using the macro in the deprecated list - we can't remove this.